# ObjectInstance
# A specific detected bounding box.

float64 bbox_width       # in [0,1] -- proportion of image width
float64 bbox_height      # in [0,1] -- proportion of image height
float64 bbox_left        # in [0,1] -- proportion of image width
float64 bbox_top         # in [0,1] -- proportion of image height
float64 confidence       # in [0,100] -- percentage
